runway/blueprints/staticsite/templates/cf_directory_index_rewrite.template.js

```diff
@@ -0,0 +1,20 @@
+'use strict';
+
+exports.handler = async function(event, context) {
+    // Extract the request from the CloudFront event that is sent to Lambda@Edge
+    var request = event.Records[0].cf.request;
+    // Extract the URI from the request
+    var olduri = request.uri;
+    // Match any '/' that occurs at the end of a URI.
+    // Replace it with a default index\n",
+    var newuri = olduri.replace(/\/$/, /{{{RewriteDirectoryIndex}}}/);
```

Comment:
`/{{{RewriteDirectoryIndex}}}/` should be `'/{{{RewriteDirectoryIndex}}}'` here (it's incorrectly adding a trailing forwardslash now)

(after changing to use the parameterized file lookup as kyle suggested, I think it'll be `'/{{RewriteDirectoryIndex}}'` (removing one of the curly braces) for its automatic parameter substitution.

Reply:
I updated this for the pre-parameterized version. Once I get an understanding of how to use the parameterized piece with it I'll update it accordingly.